### ISSUE TITLE
How does `bundletool build-apks` generate the versionCode for the APKs?

### ISSUE THREAD

**Issue by fabiendem** [google/bundletool#348]
Hi,

I want to use the command `bundletool build-apks --bundle=/MyApp/my_app.aab --output=/MyApp/my_app.apks` but it's not clear to me which `versionCode` will be generated for those APKs.
There is parameter specified at https://developer.android.com/tools/bundletool#generate_apks

Afaik, when you generate the different APKs, each should have a different `versionCode`, see https://developer.android.com/google/play/publishing/multiple-apks#VersionCodes

Can one explain how it works while using the bundletool?
I browsed the source code but I couldn't find anything useful to answer.

If they all have the same `versionCode`, what's the recommended way to edit this once I have the APKs?

Thank you!

**Comment by fabiendem** [comment]
Thank you!

Thanks for linking the doc, I missed that.
I realised that my mental model was wrong after watching those videos as well:

- https://youtu.be/IPLhLu0kvYw
- https://www.youtube.com/live/fnsxImWDxj0
- https://youtu.be/bViNOUeFuiQ

I am transitioning from APK to AAB on an existing app.

I have used a large number for one of the Abis `versionCode` (different multiplier for each abi as per the doc at https://developer.android.com/build/configure-apk-splits#configure-APK-versions), so I am assuming that when I switch to AAB, the next `versionCode` will have to be larger than the largest `versionCode` previously used.
